io-engine/src/core/env.rs

```diff
@@ -183,7 +183,7 @@ pub struct MayastorCliArgs {
     )]
     /// Persistent store timeout.
     pub ps_timeout: Duration,
-    #[clap(long = "ps-retries", default_value = "30")]
+    #[clap(long = "ps-retries", default_value = "255")]
     /// Persistent store operation retries.
     pub ps_retries: u8,
```

Comment:
Should this be u16 or u32? isn't u8 is kind of low?

Reply:
We can go as high as 255 in this case.

Reply:
u16 should be ok to use here.

Reply:
Yeah maybe use u16

Reply:
Done.

Reply:
https://github.com/openebs/mayastor-extensions/pull/592